components/Profile/SettingsModal.jsx

```diff
@@ -179,18 +204,86 @@ const SettingsModal = (props) => {
                   }}
                   maxLength={3}
                 />
-                <SettingLabel disabled={!settingState.enableRestNotifications}>Secs</SettingLabel>
+                <SettingLabel disabled={!settingState.enableRestNotifications} style={{ marginTop: 8 }}>Secs</SettingLabel>
               </View>
             </TwinView>
 
             <SubTitle>Color Theme</SubTitle>
-            <ColorButton selected={selectedColor === 'aqua'} onPress={() => { updateColorTheme('aqua'); }}>
-              <Image source={require('../../assets/blue.png')} />
-            </ColorButton>
-            <ColorButton selected={selectedColor === 'default'} onPress={() => { updateColorTheme('default'); }}>
-              <Image source={require('../../assets/purple.png')} />
-            </ColorButton>
+            <TwinView>
+              <ColorButton
+                selected={settingState.colorTheme === 'aqua'}
+                onPress={() => {
+                  setSettingState({
+                    ...settingState,
+                    colorTheme: 'aqua',
+                  });
+                }}
+                style={{ width: '45%', height: 32 }}
```

Comment:
If we need to get rid of more duplicate code could put this style into the Styled component thingy